zigup.zig

```diff
@@ -603,7 +631,28 @@ fn printDefaultCompiler(allocator: Allocator) !void {
     const default_compiler_opt = try getDefaultCompiler(allocator);
     defer if (default_compiler_opt) |default_compiler| allocator.free(default_compiler);
     const stdout = std.io.getStdOut().writer();
+    var found_master = false;
     if (default_compiler_opt) |default_compiler| {
+        if (!found_master) {
```

Comment:
Hmm, wouldn't `found_master` always be false here??

Reply:
On start, its false, and !false == true. As long as it hasn't found the master, the if condition is true. It would becomes false at https://github.com/iddev5/zigup/blob/better-default/zigup.zig#L652